### ISSUE TITLE
getExifOrientation does not work for some Uris

### ISSUE THREAD

**Issue by JonasPrap** [davemorrissey/subsampling-scale-image-view#227]
My source Uri for image file is:

`file:/storage/emulated/0/Android/data/com.***/files/private-pictures/JPEG_20160720_113440_54943315.jpg`

But it does not get orientation because of this code in library: 

`if(sourceUri.startsWith("file:///") && !sourceUri.startsWith("file:///android_asset/"))`

Notice the additional // used in the if statement.

I constructed the Uri by using this:
`imageViewTouch.setImage(ImageSource.uri(Uri.parse(file.toURI().toString())));`

**Comment by JonasPrap** [comment]
Another related issue is when I try using this:

`imageViewTouch.setImage(ImageSource.uri(file.toURI().toString()));`

Uri is not parsed correctly and I get this exception:

`java.io.FileNotFoundException: /file:/storage/emulated/0/Android/data/com.intermedix.eics/files/private-pictures/JPEG_20160720_114307_-1897635745.jpg`

Note that 'file' is from java.io.

**Comment by JonasPrap** [comment]
Solved this by: 
        imageViewTouch.setImage(ImageSource.uri(file.getAbsolutePath()));

**Comment by ShreyashPromact** [comment]
@JonasPrap 

Any idea if I want to load image from url and then apply scale effect as like this library?

Sincerely,
Shreyash

**Comment by JonasPrap** [comment]
@ShreyashPromact 
I'm not sure I completely understand the question, it seems unrelated to this issue. I think you need to download the file and save it to file system, then use this library to open that file. To accomplish that I used Retrofit2 (https://futurestud.io/blog/retrofit-2-how-to-download-files-from-server), but there are other solutions to download files and save them to disk.

**Comment by ShreyashPromact** [comment]
@JonasPrap 
Thanks for quick reply. Here in my project, I have so many images on server. I am displaying that image in the app. Now, I want to add your library as I have zoom in image functionality in the app.

But as the image coming from server, I can't use your library. Even its normal thing that No user will store such big size of image in device for there app. So, is there any support available for such condition where I can direct display image from the server (URL of the image)?

Thanks.

Sincerely,
Shreyash

**Comment by JonasPrap** [comment]
@ShreyashPromact 
First, this is not my library :) Just reported an issue in it when I started using it.

This library requires you to store image in a file....

If the images are not too huge and you are not running into memory problems I suggest trying Picasso library for image loading and this library or ImageViewZoom library for an imageView with zoom/pan capability.

**Comment by ShreyashPromact** [comment]
@JonasPrap 

You are right. I was doing same thing. I have used glide to load image. But i want to apply zoom on same loaded imageview. While here it is view. So glide is not working as I want. Same thing will be happen with Picasso. So, I have requested to have support for that.

Thanks for your reply and support. Let me know if you have any trick to load image from url and apply such zoom effect on same.

Thanks.

Sincerely,
Shreyash
